lock/src/main/java/org/infinispan/lock/impl/ClusteredLockModuleLifecycle.java

```diff
@@ -66,11 +66,14 @@ public void cacheManagerStarted(GlobalComponentRegistry gcr) {
       ClusteredLockManagerConfiguration config = extractConfiguration(gcr);
       GlobalConfiguration globalConfig = gcr.getGlobalConfiguration();
 
-      internalCacheRegistry.registerInternalCache(CLUSTERED_LOCK_CACHE_NAME, createClusteredLockCacheConfiguration(config, globalConfig),
-            EnumSet.of(InternalCacheRegistry.Flag.EXCLUSIVE));
-
-      CompletableFuture<CacheHolder> future = startCaches(cacheManager);
-      registerClusteredLockManager(gcr, future, config);
+      if (globalConfig.isClustered()) {
+         internalCacheRegistry.registerInternalCache(CLUSTERED_LOCK_CACHE_NAME, createClusteredLockCacheConfiguration(config, globalConfig),
+               EnumSet.of(InternalCacheRegistry.Flag.EXCLUSIVE));
+         CompletableFuture<CacheHolder> future = startCaches(cacheManager);
+         registerClusteredLockManager(gcr, future, config);
+      } else {
+         log.info("Configuration is not clustered, clustered locks are disabled");
```

Comment:
Info messages must be moved to the Log class

Reply:
Good catch 👍

Reply:
You mean @tristantarrant adding 
```
 @Message(value = "Configuration is not clustered, clustered locks are disabled", id = 29009)
   String configurationNotClustered();
```
and then 
```log.info(log.configurationNotClustered()```

Reply:
Correct, however you don't invoke the log.info or return a String. Method is just void and you invoke it.

```java
@LogMessage(level=INFO)
@Message(value = "Configuration is not clustered, clustered locks are disabled", id = 29009)
void configurationNotClustered();
```

```

Reply:
ok, perfect. I was thinking it was that, but I had a doubt! 👍